### ISSUE TITLE
reason for symmetric priors

### ISSUE THREAD

**Issue by vasishth** [vasishth/SchoknechtVasishthInterference#46]
>  This was necessary because we added trial id as a predictor as was suggested by a reviewer.

i don't understand this explanation.

**Comment by vasishth** [comment]
is this true? pls cite the page no.

> It is not possible to include symmetrical and directional priors within a single brms model (Bu ̈rkner, 2021), thus all priors were changed to symmetrical ones.

if i were a reviewer, for me this would be a strange reason. i would have just fit it in stan then.

wasn't the reason that we are using savage dickey to figure out the van dyke bfs and using a truncated prior there would not make sense as that was the first study on this design? our truncated priors were based on van dyke claims. so for consistency we used symmetric priors everywhere?

**Comment by vasishth** [comment]
we also switched to savage dickey to compute van dyke bfs no?

**Comment by PSchoknecht** [comment]
complete revised response:

Yes, as we had explained in the response to reviewers on August 29, 2024, all analyses
were changed in the first round of revisions. For a fair comparison of our results to the
results of the previous studies, especially Van Dyke (2007), it was necessary to compute
Bayes factors using the Savage-Dickey ratio (in the original version of the manuscript we
had used bridge sampling). Furthermore, for the Van Dyke (2007) data, the first on this
design, it only made sense to use symmetrical priors which are agnostic regarding the
sign of the effects. For consistency, we then used symmetrical priors for all our analyses
(compared to the directional priors which we had used originally).
These changes did not change the results qualitatively, but led to some numerical
changes which probably piqued the reviewer’s interest.
